src/components/ec-alert/ec-alert.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div
+    v-show="open"
+    class="ec-alert"
+    :class="typeClass"
+  >
+    <ec-icon
+      v-if="dismissable"
+      class="ec-alert__dismiss-icon"
+      :class="`ec-alert__dismiss-icon--${type}`"
+      name="simple-close"
+      :size="16"
+      @click="$emit('change', !open)"
+    />
+    <ec-icon
+      class="ec-mr--16 ec-mt--4 ec-mb--4"
```

Comment:
No utility classes in components. Add these margins to `<style>`. Utility classes are using `!important` and it won't be overridable anywhere then. Utility classes should be used only in overriding page specific designs.

Reply:
I didn't know it thanks!